lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -238,15 +238,26 @@ defmodule LivebookWeb.SessionLive.CellComponent do
   end
 
   defp cell_body(assigns) do
+    container_class = if length(Access.get(assigns, :comments, [])) >= 1, do: "h-fit", else: ""
```

Comment:
Note: all variables we access in the template should be inside assigns, so we need to use `assign` or `assign_new` in those cases.

Also, since we use this in a single place, you can just do `class={"... #{if(@comments != [], do: "h-fit")}"}`

Reply:
fixed in 7110943

for my personal learning: why should all variables be inside assigns?

Reply:
See LV docs [here](https://hexdocs.pm/phoenix_live_view/assigns-eex.html#pitfalls), specifically the `render` example :)

Reply:
thanks ! great learning